docs/zkapps/advanced-snarkyjs/actions-and-reducer.mdx

```diff
@@ -99,6 +99,42 @@ this.reducer.getActions({ fromActionsHash?: Field, endActionsHash?: Field }): A[
 
 Right now, `getActions` is available for [testing with `LocalBlockchain`](/zkapps/how-to-test-a-zkapp).
 
+### Fetching Actions from an Archive Node
+
+Since are emitted action data is stored in transaction data, actions will only be persisted by an Archive Node. To fetch action data from an Archive Node, you can use the `archive` property on the `Mina.Network` function. For example:
```

Comment:
Grammar on first sentence.

Is the `archive` parameter optional if their app doesn't need to consume this, or will it error?

> If your smart contract uses actions, you'll need to provide an `archive` property on your object passed to `Mina.Network()`, containing an Archive Node API GraphQL URL.

Reply:
So I interpret this as saying it's an optional property on the object used by `Mina.network()`, but the two methods you mentioned will fail at either build or runtime in that scenario. Is that correct? If so, can you create an issue on the Custom ESLint Plugin to identify if this scenario exists, so we can warn about it? Goal is to give devs feedback earlier.